recipes/cyavro/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "cyavro" %}
+{% set version = "0.6.2" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+   fn: {{ name }}-{{ version }}.tar.gz
+   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+   sha256: 780c164be6ed4be56f31f7c06ba46c13e3b0968f92824aa636a17e21a07d9e2d
+
+build:
+  number: 0
+  skip: True  # [win or osx]
```

Comment:
Currently cyavro doesn't build on OSX due to requiring certain kernel features that don't exist (fmemopen is used by some parts of the library).
I'm in the process of adding them to a future 0.7 release but that still needs some time to work through issues.